problems/loops.js

```diff
@@ -112,67 +201,117 @@ function endsWithFiveForLoop() {}
  */
 /*
 let i = 5;
-
+ 
 while (i > 3) {
     i += 1
 }
 */
 
 /**
  * // Write your explanation here!
- *
- */
-
- /**
- * Returns an array with all the numbers from all the numbers from 1 to 100.
- * For numbers divisible by 3, add "Fizz" instead of the number into the array.
- * For numbers divisible by 5 (and not 3), add "Buzz" instead of the number into the array.
-
- * @returns {Array} [1, 2, "Fizz", 4, "Buzz"... 98, "Fizz", "Buzz"]
-
+// infinite loop:  
+// since i is already greater than 3 there is no condition that stops the loop from running.
  */
-function fizzBuzzPart1() {}
 
+/**
+* Returns an array with all the numbers from all the numbers from 1 to 100.
+* For numbers divisible by 3, add "Fizz" instead of the number into the array.
+* For numbers divisible by 5 (and not 3), add "Buzz" instead of the number into the array.
+ 
+* @returns {Array} [1, 2, "Fizz", 4, "Buzz"... 98, "Fizz", "Buzz"]
+ 
+*/
+function fizzBuzzPart1() {
+  let arr = []
+  let num = 1
+  for (num; num <= 100; num++) {
+    if (num % 3 === 0) {
+      arr.push("Fizz")
+    }
+    else if (num % 5 === 0 && num % 3 !== 0) {
```

Comment:
No need to have && on the right side because if it's divisible by 3 the first if will be satisfied.